apps/journeys-admin/src/components/TemplateView/TemplateTags/getParentTagsWithIcon/getParentTagsWithIcon.tsx

```diff
@@ -0,0 +1,64 @@
+import { ReactNode } from 'react'
+
+import Calendar4Icon from '@core/shared/ui/icons/Calendar4'
+import Grid1Icon from '@core/shared/ui/icons/Grid1'
+import Hash2Icon from '@core/shared/ui/icons/Hash2'
+import MediaStrip1Icon from '@core/shared/ui/icons/MediaStrip1'
+import SmileyNeutralIcon from '@core/shared/ui/icons/SmileyNeutral'
+import UsersProfiles2Icon from '@core/shared/ui/icons/UsersProfiles2'
+
+import { GetTags_tags as Tag } from '../../../../../__generated__/GetTags'
+
+export interface ParentTagWithIcon {
+  id: string
+  name: string
+  icon: ReactNode
+}
+
+export function getParentTagsWithIcon(parentTags?: Tag[]): ParentTagWithIcon[] {
+  const parentTagsWithIcons: ParentTagWithIcon[] = []
+
+  parentTags?.forEach((tag) => {
+    const name = tag.name.find((name) => name.language.id === '529')?.value
```

Comment:
```suggestion
    const ENGLISH_LANGUAGE_ID = '529'
    const name = tag.name.find((name) => name.language.id === ENGLISH_LANGUAGE_ID)?.value
```

https://eslint.org/docs/latest/rules/no-magic-numbers i really like this rule. perhaps we should enforce it. The problem here is that you just have to magically know that 529 is english.